website/docs/cloud-docs/users-teams-organizations/organizations/index.mdx

```diff
@@ -152,6 +152,10 @@ HCP Terraform can perform automatic health assessments in a workspace to assess
 
 You can enforce health assessments for all eligible workspaces or let each workspace opt in to health assessments through workspace settings. Refer to [Health](/terraform/cloud-docs/workspaces/health) in the workspaces documentation for more details.
 
+#### Runs
+View all the current runs from your workspaces in HCP Terraform.
+From this page, you can delete a current run from different workspaces, filtering by `Needs Attention`, `Running` or `On Hold`.
+All the runs can also be sorted by `Latest change` or alphabetically by `Workspace/Project`.
```

Comment:
```suggestion
From the workspace home page, click **Settings** in the sidebar, then **Runs**  to view existing runs in all workspaces. You can perform the following actions:

- Click **Needs Attention** to filter runs that are ?
- Click **Running** to filter for runs that are in progress. 
- Click **On Hold** to filter runs that have paused. 
- Click on a column header to display runs in alphanumeric order according to the most recent change or workspace name.  

Refer to [Run States and Stages](/terraform/cloud-docs/run/states) for additional information about understanding run states.
```
I think we should be a little more concrete in describing how to get to this page and how to operate it. It's pretty intuitive, but framing it this way enables us to provide a little more information about the different states. Speaking of which, what does "Needs Attention" mean? Can we provide at least one example of why a run might need attention? I did not see it listed on the Run States and Stages page, but maybe I'm linking to the wrong page? We should also provide at least one possible reason why a run is in an On Hold state.

Reply:
Thanks for the quick review @trujillo-adam 

I understand your questions and concerns, but I have a couple of considerations:

1. This piece of documentation is already part (as it is just a new h4) of Organizations Settings, that is why explaining on this specific docs how to get to the page, does not make much sense here.

Regarding `needs attention`, you can read about it in different places, but here is
[a very good example]:(https://developer.hashicorp.com/terraform/cloud-docs/api-docs/notification-configurations)
`Needs Attention - "run:needs_attention" -
A plan has changes and Terraform requires user input to continue. This input may include approving the plan or a [policy override](https://developer.hashicorp.com/terraform/cloud-docs/run/states#the-policy-check-stage).`

I agree with your suggestions, but I am just a little bit concern we differ a lot from the current documentation for that page. I just want to make it smooth and more fluid as possible.

Reply:
Thank, I updated the suggestion to provide a description for "Needs Attention". I missed that there is a notifications reference in the API section of the docs. I left the link to the Run States and Stages page, though, because I did not see "On Hold" or "Running" on the notifications API reference. 

As far as providing instructions on how to get to the Runs page, I don't think we can assume that the user started at the top of the page and is working their way down and following along in the UI. Assuming users have found the correct page, they are probably using control-f to find specific references to the "run page" to learn more about it. It's fine if this section is a little different from the others. At some point, we are going to reorganize and refresh all of the content.

Reply:
@trujillo-adam just two considerations on your latest suggestion:

"From the workspace home page" if we are using this here, we need to make sure we use **workspaceS** in plural, since we are actually accessing Organization Settings and not a workspace settings. This can be a bit confusing, so we need to make sure to use Workspaces.

"to view existing runs in all workspaces" - we need to specifically use _current runs_, instead of existing runs. The difference is that we are not showing existing runs, we are only showing the Current run for each of the workspaces, and these are  different concepts.

Reply:
Thank you for clarifying! I updated the suggestion accordingly.